pkg/controller/woopra_test.go

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/codeready-toolchain/registration-service/test"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestWoopraSuite struct {
+	test.UnitTestSuite
+}
+
+func TestRunWoopraSuite(t *testing.T) {
+	suite.Run(t, &TestAuthConfigSuite{test.UnitTestSuite{}})
+}
+
+func (s *TestAuthConfigSuite) TestWoopraHandler() {
```

Comment:
I guess that this is a wrong copy-paste - you should use the `TestWoopraSuite` here, not `TestAuthConfigSuite`

Reply:
yep, fixed here: fd6538e88bf12f20ad946a8f8d04acebd5acbb02